src/components/details-panel/_details-panel.scss

```diff
@@ -0,0 +1,112 @@
+.ons-details-panel {
+    $root: &;
+
+    background-color: var(--ons-color-info-tint);
+
+    &__info-icon {
+        background: var(--ons-color-white);
+        border-radius: 50%;
+        color: var(--ons-color-ocean-blue);
+        font-size: 1.5rem;
+        font-weight: 700;
+        line-height: 2rem;
+        height: 2rem;
+        width: 2rem;
+        text-align: center;
+    }
+
+    &__banner {
+        list-style: none;
+        background-color: var(--ons-color-ocean-blue);
+        color: var(--ons-color-white);
+        cursor: pointer;
+        border: 1px solid var(--ons-color-ocean-blue);
+    }
+
+    &__banner-contents {
+        display: flex;
+    }
+
+    &__banner-title {
+        margin: 0;
```

Comment:
It looks like this might be overridden by the inline utility class `ons-u-mb-2xs` so may not be needed?